drivers/usb/device/usb_dc_sam0.c

```diff
@@ -254,6 +270,8 @@ int usb_dc_attach(void)
 	regs->CTRLA.bit.ENABLE = 1;
 	usb_sam0_wait_syncbusy();
 	regs->CTRLB.bit.DETACH = 0;
+	/* FIXME:  Should the state be changed to CONNECTED here */
```

Comment:
Actually description of this even is weird, to be connected and "enumeratiaon is completed" are different states. Use it or remove this comments (I suggest to remove).

Reply:
Thanks for the feedback.  I intended the comment to get solicit feedback since I was uncertain if the state change should be included here.  I will remove it.

Reply:
:arrow_up: still not addressed

Reply:
I'll deal with this when I get the other issue resolved and do them both at the same time.